### PR TITLE
Add toggle to remove  thumbs up/down in feature selections

### PR DESCRIPTION
**Pull request by CasDragon**
Tomek in Discord asked, and it looked simple enough. Until I realized I'd have to add a setting, and put the setting into  a menu, and then update the localization file.... 
Haven't tested, but the patch is so simple I assume it works as intended. Tomek said they'll let me know.

### CODE REVIEW

**Comment by CasDragon** [comment]
> Update to patch every Recommendation component. Still doesn't work on locked/unavailable Features, but I'm not sure that's important?

I did test this time, and it appears to function correctly to me, except on locked Features.

**Comment by CasDragon** [comment]
Oh, is this enough to warrant a new version? I can update both then

**Comment by xADDBx** [comment]
Info.json too.

Narria will publish a Nexus release tomorrow or so; so it's better to just push it out now and include it.

Also since I'm not changing the repository.json user won't be aware of the new update, so it wouldn't really matter to those. It's a good way to replace the preview versions we did in the past.

File: ToyBox/classes/MainUI/LevelUp.cs [hunk 1]
```diff
@@ -84,6 +84,11 @@ public class LevelUp {
                     Space(25);
                     Label(("Experimental".cyan() + ": lets you select any feat ignoring prerequisites.".green()).localize());
```
Comment (xADDBx) [inline ToyBox/classes/MainUI/LevelUp.cs:85]: Unrelated, but I really don't like the current localization system.

Just removing the "Experimental" would create a new key in the loc file; removing old ones is only possible by manually removing them. Keeping track of loc files across multiple versions is also a pain.

It'd probably be better to have loc strings either be static members of a localization class or something.

**Review by xADDBx — APPROVED** [review]

**Pull request merged**